NagiosEnterprises/nagioscore
# Update Exfoliation Theme for Core Services Platform

## To Test
Switch to this branch
```
./configure && make install-html
```
Does it look nice? Notice any issues?
